dev/com.ibm.ws.ejbcontainer.singleton.startup_fat/test-applications/SingletonPropertyTest.war/src/singleton/property/web/AllowWorkNoStartupServlet.java

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************************
+ * Copyright (c) 2006, 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package singleton.property.web;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.servlet.annotation.WebServlet;
+
```

Comment:
```suggestion

import org.junit.Test;
import org.junit.Ignore;

```

Only useful if previous suggestion is taken.